### DISCUSSION TITLE
Pass Keycloak frontend value to backend

### DISCUSSION THREAD

**Discussion by tiagogalvao7** [keycloak/keycloak#35661] [Q&A]
Hi guys, i'm working with Keycloak to incorporate information about privacy risks, and I would like to know if it's possivble to pass the % value in the picture, to the backend to apply authorization policies of Keycloak. This value it's requestested from an API and pass for the ftl file.
![2024-12-05 13_15_17-Login](https://github.com/user-attachments/assets/36b446b2-5b8f-42e3-9847-0d626dfe6bf2)

**Comment by codespearhead** [comment]
1. Should the percentage retrieved from the API at the time the user grants permissions to the service be stored in Keycloak’s database?
2. What should happen if the API is unavailable? Should the user still be allowed to proceed with authentication?
3. What should happen if the API doesn’t return a percentage because the service (e.g., Service A) hasn’t been evaluated? Should the user still be allowed to proceed with authentication?

**Reply by tiagogalvao7** [reply]
Hello, sorry for the late reply, but I forgot to respond. I asked this question assuming that the % value is always passed to the frontend. It's possible to pass this value to the Keycloak backend, and apply a policies regarding this value?
